### ISSUE TITLE
[Feature Request] chat/dmmessage event

### ISSUE THREAD

**Issue by eduzappa18** [mohabhassan/dgamexwrapper#4]
A new event similar to the 'servercommand' event
Something like this:
```d
dmmessage local.player local.type local.text:
       // your code here
end

local.player - Player that has sent the message
local.type   - The type of chat
               -1 = Team chat ('sayteam' command)
                0 = Global chat ('say' command)
                1 = Private chat ('sayprivate', 'sayone' command)
local.text   - The message's text

```
and as you said,  If you return 1 the message won't pass, but if you return 0 it will.

**Comment by mohabhassan** [comment]
Alright, will be there next release !
Stay sharp !

**Comment by mohabhassan** [comment]
Implemented with release v1.1.0!
